Plugin/AddressIntercept.php

```diff
@@ -9,6 +9,12 @@
 
 class AddressIntercept
 {
+    private $clientManager;
+
+    private $sailthruSettings;
```

Comment:
Thanks for covering the other PHP files as well, I assume there was only one more with this pattern that we found in CartIntercept.php